doc/whats_new.rst

```diff
@@ -1569,168 +1569,168 @@ People
+ *   8  eltermann
+ *   8  bwignall
+ *   7  Ankit Agrawal
+ *   7  CJ Carey
+ *   6  Daniel Nouri
+ *   6  Chen Liu
+ *   6  Michael Eickenberg
+ *   6  ugurthemaster
+ *   5  Aaron Schumacher
+ *   5  Baptiste Lagarde
+ *   5  Rajat Khanduja
+ *   5  Robert McGibbon
+ *   5  Sergio Pascual
+ *   4  Alexis Metaireau
+ *   4  Ignacio Rossi
+ *   4  Virgile Fritsch
+ *   4  Sebastian Saeger
+ *   4  Ilambharathi Kanniah
+ *   4  sdenton4
+ *   4  Robert Layton
+ *   4  Alyssa
+ *   4  Amos Waterland
+ *   3  Andrew Tulloch
+ *   3  murad
+ *   3  Steven Maude
+ *   3  Karol Pysniak
+ *   3  Jacques Kvam
+ *   3  cgohlke
+ *   3  cjlin
+ *   3  Michael Becker
+ *   3  hamzeh
+ *   3  Eric Jacobsen
+ *   3  john collins
+ *   3  kaushik94
+ *   3  Erwin Marsi
+ *   2  csytracy
+ *   2  LK
+ *   2  Vlad Niculae
+ *   2  Laurent Direr
+ *   2  Erik Shilts
+ *   2  Raul Garreta
+ *   2  Yoshiki Vázquez Baeza
+ *   2  Yung Siang Liau
+ *   2  abhishek thakur
+ *   2  James Yu
+ *   2  Rohit Sivaprasad
+ *   2  Roland Szabo
+ *   2  amormachine
+ *   2  Alexis Mignon
+ *   2  Oscar Carlsson
+ *   2  Nantas Nardelli
+ *   2  jess010
+ *   2  kowalski87
+ *   2  Andrew Clegg
+ *   2  Federico Vaggi
+ *   2  Simon Frid
+ *   2  Félix-Antoine Fortin
+ *   1  Ralf Gommers
+ *   1  t-aft
+ *   1  Ronan Amicel
+ *   1  Rupesh Kumar Srivastava
+ *   1  Ryan Wang
+ *   1  Samuel Charron
+ *   1  Samuel St-Jean
+ *   1  Fabian Pedregosa
+ *   1  Skipper Seabold
+ *   1  Stefan Walk
+ *   1  Stefan van der Walt
+ *   1  Stephan Hoyer
+ *   1  Allen Riddell
+ *   1  Valentin Haenel
+ *   1  Vijay Ramesh
+ *   1  Will Myers
+ *   1  Yaroslav Halchenko
+ *   1  Yoni Ben-Meshulam
+ *   1  Yury V. Zaytsev
+ *   1  adrinjalali
+ *   1  ai8rahim
+ *   1  alemagnani
+ *   1  alex
+ *   1  benjamin wilson
+ *   1  chalmerlowe
+ *   1  dzikie drożdże
+ *   1  jamestwebber
+ *   1  matrixorz
+ *   1  popo
+ *   1  samuela
+ *   1  François Boulogne
+ *   1  Alexander Measure
+ *   1  Ethan White
+ *   1  Guilherme Trein
+ *   1  Hendrik Heuer
+ *   1  IvicaJovic
+ *   1  Jan Hendrik Metzen
+ *   1  Jean Michel Rouly
+ *   1  Eduardo Ariño de la Rubia
+ *   1  Jelle Zijlstra
+ *   1  Eddy L O Jansson
+ *   1  Denis
+ *   1  John
+ *   1  John Schmidt
+ *   1  Jorge Cañardo Alastuey
+ *   1  Joseph Perla
+ *   1  Joshua Vredevoogd
+ *   1  José Ricardo
+ *   1  Julien Miotte
+ *   1  Kemal Eren
+ *   1  Kenta Sato
+ *   1  David Cournapeau
+ *   1  Kyle Kelley
+ *   1  Daniele Medri
+ *   1  Laurent Luce
+ *   1  Laurent Pierron
+ *   1  Luis Pedro Coelho
+ *   1  DanielWeitzenfeld
+ *   1  Craig Thompson
+ *   1  Chyi-Kwei Yau
+ *   1  Matthew Brett
+ *   1  Matthias Feurer
+ *   1  Max Linke
+ *   1  Chris Filo Gorgolewski
+ *   1  Charles Earl
+ *   1  Michael Hanke
+ *   1  Michele Orrù
+ *   1  Bryan Lunt
+ *   1  Brian Kearns
+ *   1  Paul Butler
+ *   1  Paweł Mandera
+ *   1  Peter
+ *   1  Andrew Ash
+ *   1  Pietro Zambelli
+ *   1  staubda
 
```

Comment:
You can play with [this](https://gist.github.com/TomDLT/1645afb6a0b2cb713416d1740d96cc4b) as a first step.